tests/test_selectors.py

```diff
@@ -106,3 +106,30 @@ def test_connection_error(self):
             get_bug.side_effect = requests.exceptions.ConnectionError
             with self.assertWarns(RuntimeWarning):
                 selectors.bug_is_fixed(None, ver)
+
+
+class SkipIfConditionTestCase(unittest.TestCase):
+    """Test :meth: `pulp_smash.selectors.skip_if`."""
+
+    class _DummyClass:  # pylint:disable=missing-docstring,too-few-public-methods
```

Comment:
We have no idea what sort of private attributes `SkipIfConditionTestCase` inherits from `unittest.TestCase`. Adding an additional private attribute should be done with care to avoid name conflicts.